clients/src/main/java/org/apache/kafka/common/requests/AbstractControlRequest.java

```diff
@@ -17,20 +17,11 @@
 package org.apache.kafka.common.requests;
 
 import org.apache.kafka.common.protocol.ApiKeys;
-import org.apache.kafka.common.protocol.types.Field;
-import org.apache.kafka.common.protocol.types.Struct;
 
 // Abstract class for all control requests including UpdateMetadataRequest, LeaderAndIsrRequest and StopReplicaRequest
 public abstract class AbstractControlRequest extends AbstractRequest {
```

Comment:
Should we make this an interface?  There seems to be nothing concrete in the class.

Reply:
It inherits from an abstract class (`AbstractRequestResponse`).

Reply:
OK.  Let's leave this alone for now.  But check out https://github.com/apache/kafka/pull/7390/files :)